Python/ceval.c

```diff
@@ -13,7 +13,7 @@
 #include "pycore_abstract.h"      // _PyIndex_Check()
 #include "pycore_call.h"          // _PyObject_FastCallDictTstate()
 #include "pycore_ceval.h"         // _PyEval_SignalAsyncExc()
-#include "pycore_code.h"          // saturating_increment()
+#include "pycore_code.h"
```

Comment:
To stay consistent with other includes here maybe let's do
```suggestion
#include "pycore_code.h"          // PyCodeObject_IncrementWarmup()
```

Reply:
I don't like those comments. It suggests that `PyCodeObject_IncrementWarmup` is the only function or struct we need, but there are several.

Reply:
Fair enough.